### PR TITLE
[#93837824] Adds blender names encoding check

### PR DESCRIPTION
**Pull request by AurL**

### CODE REVIEW

File: blender-2.5/exporter/osg/osgdata.py [hunk 1]
```diff
@@ -157,6 +157,62 @@ def findArmatureObjectForTrack(track):
...
+
+def resolve_misencoded_names(scene):
+    ''' Replace misencoded object names to avoid errors '''
+    scene = bpy.context.scene
+    renamed_count = 0
+    for obj in scene.objects:
+        try:
+            obj.name
+        except UnicodeDecodeError:
+            obj.name = 'renamed'
```
Comment (mrchlblng) [inline blender-2.5/exporter/osg/osgdata.py]: `obj.name = 'rename_{}'.format(renamed_count)` ?

File: blender-2.5/exporter/osg/osgdata.py [hunk 2]
```diff
@@ -157,6 +157,62 @@ def findArmatureObjectForTrack(track):
...
+            renamed_count += 1
+        for mod in obj.modifiers:
+            try:
+                mod.name
+            except UnicodeDecodeError:
+                mod.name = 'renamed_modifier'
```
Comment (mrchlblng) [inline blender-2.5/exporter/osg/osgdata.py]: no count here?

**Comment by cedricpinson** [comment]
what help do you need ?

**Comment by AurL** [comment]
It's okay, I put it since we didn't have the "Need review" label.
@mrchlblng can you do a quick review when you have time?

**Comment by mrchlblng** [comment]
Good for me

**Pull request merged**